src/plot_erpgrid.jl

```diff
@@ -0,0 +1,87 @@
+"""
+    function plot_erpgrid!(f::Union{GridPosition, Figure}, data::Matrix{<:Real}, pos::Vector{Point{2,Float}}; kwargs...)
+    function plot_erpgrid(data::Matrix{<:Real}, pos::Vector{Point{2,Float}}; kwargs...)
+
+Plot an ERP image.
+## Arguments:
+- `f::Union{GridPosition, Figure}`: Figure or GridPosition that the plot should be drawn into;
+- `plotData::Matrix{<:Real}`: Data for the plot visualization;
+- `pos::Vector{Point{2,Float}}`: electrode positions.
+        
+## Keyword Arguments
+- `chanNum` (Number, `30`) - Number of channels.
+- `drawLabels` (bool, `false`) - Draw channels labels over each waveform. 
+- `times`: (Vector, `1:size(plotData, 2)`) - Vector of size()
+
+## Return Value:
+The input `f`
+"""
+
+# no figure?
+plot_erpgrid(plotData::Matrix{<:Real}, pos; kwargs...) =
+    plot_erpgrid!(Figure(), plotData, pos; kwargs...)
+
+function plot_erpgrid!(
+    f::Union{GridPosition,Figure},
+    plotData::Matrix{<:Real},
+    pos;
+    chanNum = 30,
```

Comment:
why do we need chanNum? we can always do size(plotData,1)

Reply:
if you want to restrict the channels to be plotted, you'd rather expect a range/vector of channels to be used, else you can only restrict up to chanNum. I would remove this functionality and let the user subset channels / positions themselves